openapi-cli/src/test/java/io/ballerina/openapi/generators/openapi/ResponseTests.java

```diff
@@ -221,6 +221,26 @@ public void testForAllReturnType() throws IOException {
         TestUtils.compareWithGeneratedFile(ballerinaFilePath, "response/response_code.yaml");
     }
 
+    @Test(description = "When the response has float return type")
+    public void returnHasFloat() throws IOException {
```

Comment:
How about renaming the test case with a more appropriate name? (i.e. `testResponseWithFloatReturnType()`)